judo-runtime-core-dao-rdbms/src/main/java/hu/blackbelt/judo/runtime/core/dao/rdbms/query/model/RdbmsResultSet.java

```diff
@@ -502,6 +500,7 @@ public String toSql(final String prefix, final boolean includeAlias, final Coerc
         }
         final boolean addDistinct = limit != null && multiplePaths && skipParents;
 
+        final RdbmsJoin firstJoin = !joins.isEmpty() ? joins.get(0) : null;
```

Comment:
Is there any reason that firstJoin inisialization is moved here? As I see this change doesn't change in the behaviour, so unneccessary.